src/SegmentedMsg.chpl

```diff
@@ -756,16 +756,12 @@ module SegmentedMsg {
     }
   }
 
-  proc convertPythonSliceToChapel(start:int, stop:int, stride:int=1): range(stridable=true) {
-    var slice: range(stridable=true);
-    // convert python slice to chapel slice
-    // backwards iteration with negative stride
-    if  (start > stop) & (stride < 0) {slice = (stop+1)..start by stride;}
-    // forward iteration with positive stride
-    else if (start <= stop) & (stride > 0) {slice = start..(stop-1) by stride;}
-    // BAD FORM start < stop and stride is negative
-    else {slice = 1..0;}
-    return slice;
+  proc convertPythonSliceToChapel(start:int, stop:int): range(stridable=false) {
+    if (start <= stop) {
+      return start..(stop-1);
+    } else {
+      return 1..0;
+    }
```

Comment:
I was confused how this would be equivalent until i realized we force `stride` to be one before we actually call this function
```Chapel
if (stride != 1) { 
    var errorMsg = notImplementedError(pn, "stride != 1"); 
    smLogger.error(getModuleName(),getRoutineName(),getLineNumber(),errorMsg);      
    return new MsgTuple(errorMsg, MsgType.ERROR);
}
var slice = convertPythonSliceToChapel(start, stop);
```

Reply:
Yeah, probably should have included the comment in the PR description, but the commit message detailed this -- https://github.com/Bears-R-Us/arkouda/pull/1553/commits/a1ad94412799c260a098e576404f93c5386074b7

Reply:
ah cool! i missed that!